modulefiles/build.wcoss_dell_p3.intel.lua

```diff
@@ -34,16 +34,16 @@ load(pathJoin("hdf5", hdf5_ver))
 netcdf_ver=os.getenv("netcdf_ver") or "4.7.4"
 load(pathJoin("netcdf", netcdf_ver))
 
-nccmp_ver=os.getenv("nccmp_ver") or "1.8.7.0"
+nccmp_ver=os.getenv("nccmp_ver") or "1.8.9.0"
 load(pathJoin("nccmp", nccmp_ver))
 
-esmf_ver=os.getenv("esmf_ver") or "8_1_0_beta_snapshot_27"
+esmf_ver=os.getenv("esmf_ver") or "8_2_0"
 load(pathJoin("esmf", esmf_ver))
 
 bacio_ver=os.getenv("bacio_ver") or "2.4.1"
 load(pathJoin("bacio", bacio_ver))
 
-g2_ver=os.getenv("g2_ver") or "3.4.1"
+g2_ver=os.getenv("g2_ver") or "3.4.2"
```

Comment:
Curious about why you upgraded the G2 version.

Reply:
In order to use `esmf_8_2_0`, I had to change `ips_ver` to `18.0.5.274`, which in return required changing `g2` to `3.4.2` and  `nccmp` to `1.8.9.0` to be able to module load.

Reply:
Ok.